satellite/modules/vm_manage.py

```diff
@@ -10,6 +10,18 @@
 
 
 def vm_create(compute_ip, name, ram, cpus, disk_size, location_url, kickstart_loc):
+    """Create virtual machine on remote system
+
+    :param compute_ip: Remote system IP address
+    :param name: Name of virtual machine
+    :param ram: RAM size for virtual machine
+    :param cpus: Number of virtual CPUS for virtual machine
+    :param disk_size: Disk size for virtual machine
+    :param location_url: URL location of OS
+    :param kickstart_loc: location of kickstart
+
+    :returns: Boolean, true if success or false
```

Comment:
change returns line to Boolean **True** if success else **False**

Reply:
use return instead of returns